style.css

```
@@ -308,6 +333,145 @@ nav {
   margin: 0 auto 10px auto;
 }
 
+.about-me {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 20px;
+  gap: 12px;
+  width: 100%;
+  height: 1451px;
+  background: #1c1a19;
+  background-image: url(images/images-self/about-me1.png);
+  background-repeat: no-repeat;
+  background-position: top 350px right;
+}
+
+.project-details1 {
+  margin-top: 5px;
+  width: 96%;
+  justify-content: center;
+  align-items: center;
+  height: 95vh;
+  position: fixed;
+  top: 10px;
+  background-color: white;
+  z-index: 1000;
+  margin-left: 6px;
+  margin-right: 2px;
+  overflow-y: scroll;
+  overflow-x: hidden;
+  backdrop-filter: blur(5px);
+}
+
+body.active-popup .head {
+  filter: blur(5px);
+}
+
+body.active-popup .home {
+  filter: blur(5px);
+}
+
+body.active-popup .about-me {
+  filter: blur(5px);
+}
+
+body.active-popup footer {
+  filter: blur(5px);
+}
+
+body.active-popup .social-icons3 {
+  filter: blur(5px);
+}
+
+.closebutton {
+  color: #67798e;
+  position: relative;
+  left: 93%;
+  top: -50px;
+  border-style: none;
+  background-color: white;
+  font-size: 27px;
+}
+
+.project-details1 h2 {
+  font-family: "Roboto Serif", serif;
+  height: 44px;
+  align-items: center;
+  font-size: 32px;
+  display: flex;
+  padding: 20px;
+  color: #172b4d;
+  margin-top: 20px;
+}
+
+.project-details1 h3 {
+  display: none;
+}
+
+.project-details1 .project-text {
+  font-family: "Inter", sans-serif;
+  font-size: 16px;
+  font-weight: 500;
+  display: flex;
+  gap: 10px;
+  padding: 20px 20px;
+  line-height: 24px;
+}
+
+.p-elements {
+  display: none;
+}
+
+.programing {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  list-style: none;
+  padding: 20px;
+  width: 90%;
+}
+
+.programing li {
+  text-decoration: none;
+  border: 1px solid #8993a4;
+  padding: 10px 12px;
+}
+
+.details-img1 {
+  margin-left: 20px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 90%;
+}
+
+.clicks {
+  display: flex;
+  flex-direction: row;
+  margin-left: 5%;
+  padding-top: 20px;
+  list-style: none;
+  gap: 20px;
+}
+
+.clicks li {
+  border: 1px #8993a4;
+  color: #fff;
+  background-color: #ff6b00;
+  padding: 12px;
+}
```

- [ ] As previous reviewer suggested, kindly make this icons bigger. Please accept my suggestions :)


```suggestion
.clicks li {
  border: 1px #8993a4;
  color: #fff;
  background-color: #ff6b00;
  padding: 8px 12px;
  display: flex;
  align-items: center;
  justify-content: center;
}
```